script.js

```diff
@@ -0,0 +1,23 @@
+function calc() {
+
+    var a = parseInt(document.querySelector("#num1").value)
+    var b = parseInt(document.querySelector("#num2").value)
```

Comment:
Names are important, remember to name things what they are. Names like `a` and `b` have no meaning.